### PR TITLE
Stax 4bpp icons + Minor wording

### PR DESCRIPTION
**Pull request by agrojean-ledger**

### CODE REVIEW

File: src/ui/address/address_ui_nbgl.c [hunk 1]
```diff
@@ -33,7 +33,7 @@ static void confirmationChoiceClbk(bool confirm) {
         nbgl_useCaseStatus("ADDRESS\nAPPROVED", true, display_idle_menu);
```
Comment (tdejoigny-ledger) [inline src/ui/address/address_ui_nbgl.c]: Please replace `ADDRESS\nAPPROVED` by `ADDRESS\nVERIFIED`

File: src/ui/main/idle_menu_nbgl.c [hunk 2]
```diff
@@ -42,19 +42,14 @@ static bool about_nav_clbk(uint8_t page, nbgl_pageContent_t* content) {
...
 }
 
 void display_idle_menu() {
-    nbgl_useCaseHome("XRP",
-                     &C_icon_XRP_64px,
-                     "This app confirms actions on\nthe XRP network.",
-                     true,
-                     display_about_menu,
-                     on_quit_clbk);
+    nbgl_useCaseHome("XRP", &C_icon_XRP_64px, NULL, true, display_about_menu, on_quit_clbk);
```
Comment (tdejoigny-ledger) [inline src/ui/main/idle_menu_nbgl.c]: `APPNAME` can be used instead of `"XRP"`

File: src/ui/address/address_ui_nbgl.c (hunk 1 above)
Comment (agrojean-ledger) [reply src/ui/address/address_ui_nbgl.c]: Done.

File: src/ui/main/idle_menu_nbgl.c (hunk 2 above)
Comment (agrojean-ledger) [reply src/ui/main/idle_menu_nbgl.c]: Done.

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**